docusaurus.config.js

```diff
@@ -162,18 +162,18 @@ const config = {
           position: 'right',
           dropdownActiveClassDisabled: true,
           dropdownItemsAfter: [
+            // {
+            //   to: 'https://0-26-0.docs.pomerium.com/docs',
+            //   label: 'v0.26 (latest)',
+            // },
```

Comment:
I find this confusing — I think it might be clearer without adding this.

Reply:
Sure, I can remove it.